python/amici/sbml_import.py

```diff
@@ -1352,53 +1352,54 @@ def assignmentRules2observables(sbml_model,
 def noise_distribution_to_cost_function(
         noise_distribution: str) -> Callable[[str], str]:
     """
-    Parse cost string to a cost function definition amici can work with.
+    Parse noise distribution string to a cost function definition amici can
```

Comment:
I think up in the file where this function is used, the assignee is also called ~n~llh ...

Reply:
yes... I shortly considered changing all, but that will be quite some places. better to be done separately. also not sure if "llh" is the best name, as it could be anything.